docs/ruleset_functions/get_fan_system_object_supply_return_exhaust_relief_total_kW_CFM.md

```diff
@@ -0,0 +1,140 @@
+# get_fan_system_object_supply_return_exhaust_relief_total_kW_CFM
+
+**Description:** Get the supply, return, exhaust, and relief total fan power, CFM, quantity, and information about whether the pressure drop is consistent across the fans if more than one for a fan system object.   The function returns a dictionary that saves the supply, return, exhaust, and relief fan power as a list, saves the supply, return, exhaust, and relief cfm as a list, saves the supply, return, exhaust, and relief quantity as a list, and saves for each fan whether the pressure drop is undefined, identical, or different across fans (if only one it will return undefined or identical) {"Fan_Power": [supply fan power kW, return fan power kW, exhaust fan power kW, relief fan power kW], "Fan_CFM": [supply fan power CFM, return fan power CFM, exhaust fan power CFM, relief fan power CFM], "Fan_Qty": [supply fan quantity, return fan quantity, exhaust fan quantity, relief fan quantity], "Fan_Pressure": [returns either "IDENTICAL", "UNDEFINED", "DIFFERENT" for supply fans, returns either "IDENTICAL", "UNDEFINED", "DIFFERENT" for return fans, returns either "IDENTICAL", "UNDEFINED", "DIFFERENT" for exhaust fans, returns either "IDENTICAL", "UNDEFINED", "DIFFERENT" for relief fans]}. Values will be equal to zero where not defined for a fan system or as otherwise specified above. For CFM values this function assumes that all fans associated with the supply_fans object are in parallel (i.e., if multiple fans the cfm is additive.) but returns the quantity and information about the pressure drop across fans to help assess whether series or parallel.
+
```

Comment:
Haven't we decided to have "undertermined" outcome if there is more than one supply, return or relief fan in the baseline? I think we should also assume that when there are multiple exhaust fans they are zonal fans.  This function may stay as is though because the corresponding logic may be implemented in the RDS.